package.json

```diff
@@ -48,6 +48,7 @@
     "react-18-blockies": "^1.0.6",
     "react-dom": "^18.2.0",
     "react-i18next": "^12.2.0",
+    "react-infinite-scroll-component": "^6.1.0",
```

Comment:
No need to install a new dependency for this. As suggested in the design we can use https://ionicframework.com/docs/api/infinite-scroll

![11072024163257](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/312e5788-faaf-4d82-9243-050f48dfa91a)

Reply:
`ion-infinite-scroll` not working on our page. 

The reason because this component listen `scroll` event from closest scroll container and in this case it is `main.inner-scroll` element (child node of `ion-content`). 

But because we are setting `overflow` property of this element is `hidden`, it make scroll event not trigger => `ion-infinite-scroll` also not working.

Reply:
I see, thanks 👍